pkg/core/interops.go

```diff
@@ -51,9 +51,7 @@ var systemInterops = []interop.Function{
 	{Name: interopnames.SystemBlockchainGetTransactionHeight, Func: bcGetTransactionHeight, Price: 1 << 15,
 		RequiredFlags: flags.ReadStates, ParamCount: 1},
 	{Name: interopnames.SystemContractCall, Func: contract.Call, Price: 1 << 15,
-		RequiredFlags: flags.AllowCall, ParamCount: 3},
-	{Name: interopnames.SystemContractCallEx, Func: contract.CallEx, Price: 1 << 15,
-		RequiredFlags: flags.AllowCall, ParamCount: 4},
+		RequiredFlags: flags.AllowCall, ParamCount: 5},
```

Comment:
Isn't it 4 now (hash, method, flags, args)?

Reply:
Fixed.